scripts/org.ooni.probe.cli.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>org.ooni.probe.daily-run</string>
+
+    <key>KeepAlive</key>
+    <false/>
+    <key>RunAtLoad</key>
+    <true/>
+
+    <key>Program</key>
+    <string>/Users/USERNAME/.ooniprobe/ooniprobe</string>
+    <key>ProgramArguments</key>
+    <key>ProgramArguments</key>
+    <array>
+        <string>--config="/Users/USERNAME/.ooniprobe/config-100sites.json"</string>
+        <string>--batch</string>
+        <string>run</string>
+    </array>
+
+    <key>StartInterval</key>
+    <integer>3600</integer>
```

Comment:
The main concern that I have is that subsequent experiments may overlap

Reply:
The should not happen if you set the limit to 100 URLs, but some more testing is probably required to validate this.

On my network, which is pretty fast, I run this consistently in under 10 minutes, but actually closer to 5 minutes.